__utils__/assert-project/src/index.ts

```diff
@@ -138,17 +138,17 @@ export function assertProject (projectPath: string, encodedRegistryName?: string
     readCurrentLockfile () {
       try {
         return readYamlFile(path.join(getVirtualStoreDir(), 'lock.yaml'))
-      } catch (err: any) { // eslint-disable-line
-        if (err.code === 'ENOENT') return null!
+      } catch (err: unknown) {
+        if (err instanceof Error && 'code' in err && err.code === 'ENOENT') return null!
```

Comment:
So this `return null!` would have type `never`, which would mean that `readCurrentLockfile` has a non-null return type despite the fact that it can return `null` in reality?

Reply:
I don't care about it in this PR. This was like this already.